### PR TITLE
add: tab bottom navigation

### PR DESCRIPTION
**Pull request by HuziMuzi**
Added tabs bottom navigation and routs to main screens

### CODE REVIEW

File: src/navigation/navigation.tsx [hunk 1]
```diff
@@ -1,32 +1,80 @@
...
 
 export type RootStackParamList = {
   [ScreenList.HOME]: undefined; // undefined because you aren't passing any params to the home screen
-  [ScreenList.EXAMPLE]: undefined;
+  [ScreenList.CREATE_TEST]: undefined;
+  [ScreenList.TESTS_LIST]: undefined;
+  [ScreenList.LIVE_CODING]: undefined;
+};
+
+const screenOptionsIcons = (routeName: string, color: string, size: number) => {
```
Comment (antoniosk10) [inline src/navigation/navigation.tsx:25]: `getIcon`

File: src/screens/Home.tsx [hunk 2]
```diff
@@ -1,26 +1,19 @@
...
 import {SafeAreaView} from 'react-native-safe-area-context';
 
-interface IHomeScreen {
-  navigation: NavigationType;
-}
-export const Home: FC<IHomeScreen> = ({navigation}) => {
-  const handlerNavigationExample = () => {
-    navigation.push(ScreenList.EXAMPLE);
-  };
+interface IHomeScreen {}
```
Comment (antoniosk10) [inline src/screens/Home.tsx:9]: ?

File: src/screens/LiveCoding.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import {StyleSheet, Text} from 'react-native';
+
```
Comment (antoniosk10) [inline src/screens/LiveCoding.tsx:3]: remove

File: src/screens/TestsList.tsx [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import {StyleSheet, Text} from 'react-native';
+
```
Comment (antoniosk10) [inline src/screens/TestsList.tsx:3]: remove

File: src/navigation/navigation.tsx [hunk 5]
```diff
@@ -1,32 +1,80 @@
...
-        <Screen name={ScreenList.EXAMPLE} component={Example} />
-      </>
-    </Navigator>
+    <Tab.Navigator
+      initialRouteName={'home'}
+      screenOptions={({route}) => ({
+        tabBarIcon: ({size, focused}) =>
+          screenOptionsIcons(
+            route.name,
+            focused ? Color.BlueLight : Color.GrayDark,
```
Comment (antoniosk10) [inline src/navigation/navigation.tsx:68]: lets add this condition into function

**Review by antoniosk10 — DISMISSED** [review]

**Review by KhaninPasha0488 — APPROVED** [review]

File: src/navigation/styles.ts [hunk 6]
```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components/native';
+import {StyleSheet} from 'react-native';
+
+export const ButtomCenter = styled.TouchableOpacity`
```
Comment (antoniosk10) [inline src/navigation/styles.ts:4]: buttom?

**Review by Julia-Kovalchuk — APPROVED** [review]

**Review by KhaninPasha0488 — DISMISSED** [review]

**Review by KhaninPasha0488 — APPROVED** [review]

**Review by Julia-Kovalchuk — APPROVED** [review]

**Pull request merged**